utils/getStates.js

```diff
@@ -49,6 +50,21 @@ module.exports = async (
 		if (!json) {
 			spinner.info(`${cyan(`Sorted by:`)} ${sortBy}${isRev}`);
 		}
+
+		const csvWriter = createCsvWriter({
+			path: 'output/chart.csv',
+			header: [
+			  {id: 'state', title: 'State'},
+			  {id: 'cases', title: 'Cases'},
+			  {id: 'todayCases', title: 'Cases (today)'},
+			  {id: 'deaths', title: 'Deaths'},
+			  {id: 'todayDeaths', title: 'Deaths (today)'},
+			  {id: 'active', title: 'Active'},
+			]
+		});
+
+		csvWriter.writeRecords(allStates);
+
```

Comment:
`corona states` throws an error when the output folder does not exist. Should create the directory if it doesn't exist before writing the records.

Reply:
Hi, I want to add code to remove this bug. How do I do this? Should I clone, make amends and then do a PR ? or something else ?

Reply:
i suggest you can clone and add a new commit to this pr with the fix, thanks